notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<p>Below you are working with a salad dataset, can you also update this example here to show something from that dataset. It will be cleaner and not require context switching</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>